bundle/EventListener/NoViewTemplateEventListener.php

```diff
@@ -70,6 +71,10 @@ public function getController(FilterControllerEvent $event): void
             return;
         }
 
+        if ($view->getControllerReference()->controller === 'Symfony\Bundle\FrameworkBundle\Controller\RedirectController::urlRedirectAction') {
```

Comment:
Can we use something like checking if the controller starts with `Symfony\Bundle\FrameworkBundle\Controller\RedirectController::`, to make sure other actions from `RedirectController` are supported too?

Reply:
Also, we should probably check if `$view->getControllerReference()` is actually an instance of `ControllerReference`, since nothing guarantees it is set in the view.